frontend/src/components/workspaces/Templates/TemplatesTable/TemplatesTable.less

```
@@ -0,0 +1,7 @@
+// WE HAVE TO FIX THIS HARD CODED HEIGHT
+@ext-height: 380px; //465px if we want to add a Box footer
+
+.cl-templates-table{
+  min-height: 171px;  
+  height: calc(100vh - @ext-height); 
+}
```

```suggestion
// document.documentElement.style.setProperty('--box-ext-height', `${value}px`);

.cl-templates-table {
  min-height: 171px;  
  height: calc(100vh - var(--box-ext-height, 380px)); 
}
```
I would rather use CSS variables so that we could easily set them on runtime through JS or css breakpoints...

---

i didn't know this way, so ok i agree. Do i commit and squash this fix?

---

That's good for me. We should jsut see if it works correctly then for me we can merge

---

ok, i proceed